services/121-service/src/registration/mappers/registration-views.mapper.ts

```diff
@@ -142,10 +143,9 @@ export class RegistrationViewsMapper {
     attribute: ProgramRegistrationAttributeEntity,
     value: unknown,
   ): unknown {
+    const english = RegistrationPreferredLanguage.en;
     const selectedOption = attribute.options?.find((o) => o.option === value);
-    if (selectedOption && selectedOption.label && selectedOption.label['en']) {
-      return selectedOption.label['en'];
-    }
-    return value;
+
+    return selectedOption?.label?.[english] ?? value;
```

Comment:
is the significance of english here actually to be fallbackLanguage? If so, we better name it as such.